data/json/items/ammo/9x18.json

```diff
@@ -0,0 +1,56 @@
+[
+  {
+    "id": "9x18mm",
+    "type": "AMMO",
+    "price": 2600,
+    "name": "9x18mm 57-N-181S",
+    "name_plural": "9x18mm",
+    "description": "9x18mm Makarov ammunition with 93gr steel core FMJ bullets.  Analogous to the western 9x19mm cartridge, the 9x18mm round was very common in the Eastern Bloc during the 20th century and remained in Russian military service into the 21st century.",
+    "weight": 7,
+    "volume": 1,
+    "count": 50,
+    "stack_size": 50,
+    "material": ["steel", "powder"],
+    "symbol": "=",
+    "color": "yellow",
+    "ammo_type": "9x18",
+    "casing": "9x18mm_casing",
+    "damage": 16,
+    "range": 13,
+    "pierce": 2,
+    "dispersion": 140,
+    "recoil": 185,
+    "effects": ["COOKOFF"]
+  },
+  {   
+    "id": "9x18mmfmj",
+    "copy-from": "9x18mm",
+    "type": "AMMO",
+    "name": "9x18mm RG028",
+    "name_plural": "9x18mm RG028",
+    "description": "9x18mm Makarov RG028 ammunition.  The RG028 round uses bullets with a hardened steel core to improve armor penetration.",
+    "weight": 8,
+    "price": 3100,
+    "relative": {
+        "damage": -2,
+        "pierce": 8
+    }
+  },
+  {   
+    "id": "9x18mmP2",
+    "copy-from": "9x18mm",
+    "type": "AMMO",
+    "name": "9x18mm SP-7",
+    "name_plural": "9x18mm SP-7",
+    "description": "9x18mm Makarov SP-7 ammunition.  Analogous to the western 9x19mm cartridge, the 9x18mm round was very common in the Eastern Bloc during the 20th century and remained in Russian military service into the 21st century.",
+    "weight": 8,
```

Comment:
If your goal is migrating the remaining guns to use magazines then thats ok. However if it's re-balancing the entire guns/ammo then that's a job I can't help you with now and shouldn't be attempted immediately prior to `0.D`.

Although in no way are you at fault this piecemeal comment/reply approach is consuming a lot of time. Try and stay on-topic and push larger commits. You'll need to learn to rebase as opening multiple similar PR's isn't the best approach.